src/installer.js

```diff
@@ -290,13 +293,15 @@ module.exports.install = async function install(deps, options, logger) {
       packageManager.options && packageManager.options.dev
         ? '--save-dev'
         : null;
+    quietOptions = [];
```

Comment:
it should be `--reporter=silent` https://pnpm.io/cli/install#--reportername

Reply:
updated